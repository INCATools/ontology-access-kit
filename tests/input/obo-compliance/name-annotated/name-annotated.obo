format-version: 1.4
ontology: name-annotated

[Term]
id: X:1
name: x1 {source="PMID:123457"}

