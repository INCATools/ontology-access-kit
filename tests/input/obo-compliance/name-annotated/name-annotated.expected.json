{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/name-annotated.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://purl.obolibrary.org/obo/X_1",
      "lbl" : "x1",
      "type" : "CLASS"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#id",
      "lbl" : "id",
      "type" : "PROPERTY"
    } ]
  } ]
}